source/extensions/omni.isaac.lab/omni/isaac/lab/managers/manager_term_cfg.py

```diff
@@ -209,6 +209,16 @@ class EventTermCfg(ManagerTermBaseCfg):
         This is only used if the mode is ``"interval"``.
     """
 
+    min_step_count_between_reset: int = 0
+    """The minimum number of environment steps between when term is applied.
+
+    When mode is "reset", the term will not be applied on the next reset unless
+    the number of steps since the last application of the term has exceeded this.
```

Comment:
```suggestion
    the number of steps since the last application of the term has exceeded this.
    This is useful to avoid calling this term too often and improve performance.
```